doc/howto/import_machines_to_instances.md

```diff
@@ -27,7 +27,7 @@ The tool can create both containers and virtual machines:
 
 * When creating a container, you must provide a disk or partition that contains the root file system for the container.
   For example, this could be the `/` root disk of the machine or container where you are running the tool.
-* When creating a virtual machine, you must provide a bootable disk, partition or image.
+* When creating a virtual machine, you must provide a bootable disk, partition, or an image in either VMDK or QCOW2 format.
```

Comment:
You can probably still provide raw format as well, or has this option been removed?

Reply:
Yes, `raw` format is still there. Thanks.